inventory/sample/group_vars/k8s-cluster/k8s-cluster.yml

```diff
@@ -105,8 +105,8 @@ kube_service_addresses_ipv6: fd85:ee78:d8a6:8607::1000/116
 # Internal network. When used, it will assign IPv6 addresses from this range to individual pods.
 # This network must not already be in your network infrastructure!
 # This is only used if enable_ipv6_dual_stack is set to true.
-# This provides room for 512 nodes with 126 pods per node.
-kube_pods_subnet_ipv6: fd85:ee78:d8a6:8607:0:0:0:2000/112
+# This provides room for 64 nodes with 126 pods per node.
+kube_pods_subnet_ipv6: fd85:ee78:d8a6:8607:0:0:0:2000/116
```

Comment:
10.233.0.0/18 gives 16382 ips
fd85:ee78:d8a6:8607:0:0:0:2000/116 gives 4096, shouldn't we use a /114 ?

Reply:
It was a few months ago now, but I believe I ran into [this limitation in Calico](https://github.com/projectcalico/node/blob/39381f0/pkg/startup/startup.go#L668) and eventually reverted to use a `/116` network size.

I'm not sure why that limitation is set in Calico however. Memory efficiency or strains on the SDN perhaps?

Reply:
This check the size of the node subnet, not the cluster subnet, is `kube_network_node_prefix_ipv6` properly used ?

Reply:
Well I can't recall why now as I've squashed the commits, but I've set this to a `/112` and `kube_network_node_prefix_ipv6` to `/120` and it's passing CI still. Netchecker seems to show IPv6 IPs on the pods too (I think)